app/sidekiq/decision_review/submit_upload.rb

```diff
@@ -1,73 +1,100 @@
                      when 'NOD'
-                       handle_notice_of_disagreement(appeal_submission, file_number_or_ssn, sanitized_file)
+                       handle_notice_of_disagreement(appeal_submission_upload, file_number_or_ssn, sanitized_file)
                      when 'SC'
                        handle_supplemental_claim(appeal_submission, file_number_or_ssn, sanitized_file)
                      else
                        raise "Unknown appeal type (#{type})"
                      end.body.dig('data', 'id')
       appeal_submission_upload.update!(lighthouse_upload_id: lh_upload_id)
-      log_success(internal_id: appeal_submission.id, lighthouse_uuid: appeal_submission.submitted_appeal_uuid,
-                  lighthouse_evidence_uuid: lh_upload_id, appeal_type: appeal_submission.type_of_appeal)
       StatsD.increment("#{STATSD_KEY_PREFIX}.success")
+    rescue => e
+      log_formatted(
+        key: :evidence_upload_to_lighthouse,
+        form_id: '10182',
+        user_uuid: appeal_submission&.user_uuid,
+        upstream_system: 'AWS S3 or Lighthouse',
+        is_success: false,
+        body: e.message, # Test 504 error as an example
+        params: {
+          appeal_submission_upload_id: appeal_submission_upload&.id
+        }
+      )
+      StatsD.increment("#{STATSD_KEY_PREFIX}.error")
+      raise e
     end
+    # rubocop:enable Metrics/MethodLength
 
     private
 
-    def log_success(internal_id:, lighthouse_uuid:, lighthouse_evidence_uuid:, appeal_type:)
-      Rails.logger.info({
-                          message: 'Appeal evidence upload complete',
-                          appeal_submission_id: internal_id,
-                          appeal_type:,
-                          lighthouse_submission: {
-                            id: lighthouse_uuid
-                          },
-                          lighthouse_evidence_upload_id: lighthouse_evidence_uuid
-                        })
+    def get_sanitized_file(appeal_submission_upload)
+      form_attachment = appeal_submission_upload.form_attachment
+      appeal_submission = appeal_submission_upload.appeal_submission
+      sanitized_file = form_attachment&.get_file
+      if appeal_submission&.type_of_appeal == 'NOD'
+        log_formatted(
+          key: :evidence_upload_retrieval,
+          form_id: '10182',
+          user_uuid: appeal_submission&.user_uuid,
+          upstream_system: 'AWS S3',
+          is_success: sanitized_file.present?,
+          params: {
+            appeal_submission_upload_id: appeal_submission_upload&.id,
+            form_attachment_id: form_attachment&.id
+          }
+        )
+      end
+      sanitized_file
     end
 
     def get_dr_svc
       DecisionReviewV1::Service.new
     end
 
-    def handle_notice_of_disagreement(appeal_submission, file_number_or_ssn, sanitized_file)
+    def handle_notice_of_disagreement(appeal_submission_upload, file_number_or_ssn, sanitized_file)
```

Comment:
Where possible, for consistency, I've added logging to the service layer, rather than the job layer. That is why you see me passing additional attributes to the service calls below.